lib/xcodeproj/project/object/build_phase.rb

```diff
@@ -279,16 +279,14 @@ class PBXShellScriptBuildPhase < AbstractBuildPhase
 
         # @return [String] the actual script to perform.
         #
-        # @note   Defaults to the empty string.
+        # @note   Defaults to a comment string.
         #
-        attribute :shell_script, String, ''
+        attribute :shell_script, String, "# Type a script or drag a script file from your workspace to insert its path.\n"
```

Comment:
I'm not sure that Xcodeproj should care about imitating this behavior of Xcode. I assume this default exists mostly for Xcode UI discoverability/usability, but isn't terribly helpful for this gem. I don't feel strongly about this though.

Reply:
Its nuanced in a sense that if xcodeproj creates an empty script phase with defaults then Xcode will change it again therefore making any repo that is tracked by version control to be dirty.

Reply:
Although yes not many people in reality create empty script phases through Xcodeproj but its OK to fix this here.

Reply:
Yes it does at least with Xcode 10. It also adds it as a comment using `#` and sets the value. Compared to Xcode 9 that just shows the same text without `#` but _does not_ set it as part of `pbxproj`. I tested on both and this is why I changed it here for Xcode 10 and going forward.